### ISSUE TITLE
[GF 16.4.5] Blame! suggestion

### ISSUE THREAD

**Issue by No3371** [skyjay1/GreekFantasy#30]
```
****************** Blame Report 1.7.6 ******************

 This ConfiguredFeature was found to be not registered. Look at the JSON info and try to
 find which mod it belongs to. Then go tell that mod owner to register their ConfiguredFeature
 as otherwise, it will break other mods or datapacks that registered their stuff.

 JSON info : {
  "config": {
    "feature": {
      "config": {
        "feature": {
          "config": {},
          "type": "greekfantasy:small_shrine"
        },
        "decorator": {
          "config": {
            "outer": {
              "config": {},
              "type": "minecraft:square"
            },
            "inner": {
              "config": {},
              "type": "minecraft:heightmap"
            }
          },
          "type": "minecraft:decorated"
        }
      },
      "type": "minecraft:decorated"
    },
    "decorator": {
      "config": {
        "chance": 125
      },
      "type": "minecraft:chance"
    }
  },
  "type": "minecraft:decorated"
}

 Biome affected : [byg:blue_taiga_hills, byg:snowy_coniferous_clearing, byg:evergreen_hills, byg:aspen_clearing, byg:twilight_valley_hills, byg:coniferous_forest_hills, minecraft:modified_gravelly_mountains,
                   byg:great_lake_isles, byg:marshlands, minecraft:swamp, byg:red_oak_forest_hills, byg:rainbow_beach, byg:jacaranda_forest, minecraft:desert_hills,
                   minecraft:dark_forest, byg:meadow, minecraft:jungle_hills, minecraft:mushroom_field_shore, byg:grassland_plateau, byg:wooded_red_rock_mountains, minecraft:giant_spruce_taiga,
                   byg:flowering_enchanted_grove, byg:red_desert, byg:deciduous_forest_hills, byg:tropical_islands, byg:tropical_fungal_forest, byg:cherry_blossom_clearing, byg:aspen_forest,
                   byg:enchanted_forest_hills, byg:frozen_lake, byg:zelkova_forest_hills, byg:white_beach, byg:guiana_clearing, byg:boreal_clearing, byg:fresh_water_lake,
                   greekfantasy:olive_forest, minecraft:mountain_edge, byg:enchanted_forest, byg:coniferous_clearing, minecraft:the_void, minecraft:wooded_hills, byg:glowing_ancient_forest,
                   byg:seasonal_taiga_hills, minecraft:stone_shore, byg:canyons, byg:boreal_forest_hills, minecraft:bamboo_jungle_hills, byg:mangrove_marshes, byg:amaranth_fields,
                   byg:blue_giant_taiga, minecraft:tall_birch_forest, minecraft:modified_wooded_badlands_plateau, byg:black_forest_hills, minecraft:eroded_badlands, byg:cika_mountains, byg:tropical_fungal_rainforest_hills,
                   byg:araucaria_savanna, byg:bayou, byg:rose_fields, byg:lush_red_desert, byg:woodlands, minecraft:plains, byg:snowy_coniferous_forest_hills,
                   minecraft:birch_forest_hills, minecraft:modified_badlands_plateau, byg:bluff_peaks, byg:maple_hills, byg:canyon_edge, byg:red_rock_mountains, byg:red_oak_forest,
                   byg:seasonal_giant_taiga, byg:dummy, byg:aspen_forest_hills, minecraft:jungle, byg:wooded_grassland_plateau, byg:boreal_forest, byg:cold_swamplands,
                   minecraft:badlands, byg:fungal_patch, byg:redwood_tropics, byg:seasonal_deciduous_forest_hills, minecraft:wooded_badlands_plateau, byg:seasonal_birch_forest_hills, byg:snowy_evergreen_clearing,
                   byg:redwood_clearing, byg:jacaranda_clearing, byg:snowy_blue_giant_taiga, byg:snowy_black_beach, minecraft:tall_birch_hills, byg:black_forest_clearing, byg:enchanted_grove,
                   minecraft:savanna_plateau, byg:deciduous_clearing, byg:redwood_mountains, minecraft:wooded_mountains, minecraft:jungle_edge, minecraft:savanna, byg:sierra_range,
                   byg:weeping_witch_clearing, byg:allium_fields, byg:snowy_rocky_black_beach, byg:bamboo_forest, byg:dunes, byg:prairie_clearing, byg:wooded_meadow,
                   minecraft:mushroom_fields, byg:flowering_meadow, byg:snowy_blue_taiga_hills, minecraft:forest, byg:alpine_foothills, byg:oasis, byg:coral_mangroves,
                   byg:flowering_ancient_forest, byg:forest_fault, minecraft:sunflower_plains, minecraft:gravelly_mountains, byg:ancient_forest, byg:sierra_valley, byg:snowy_deciduous_forest_hills,
                   byg:grove, byg:ebony_hills, minecraft:flower_forest, byg:tropical_rainforest_hills, minecraft:shattered_savanna_plateau, minecraft:desert, minecraft:beach,
                   byg:seasonal_forest_hills, byg:guiana_shield, byg:pointed_stone_forest, byg:red_rock_highlands, byg:mojave_desert, byg:cherry_blossom_forest, byg:glowshroom_bayou,
                   byg:red_desert_dunes, minecraft:shattered_savanna, byg:seasonal_deciduous_forest, byg:autumnal_valley, byg:stone_forest, byg:crag_gardens, byg:prairie,
                   minecraft:desert_lakes, byg:snowy_deciduous_clearing, byg:baobab_savanna, minecraft:bamboo_jungle, byg:araucaria_forest, minecraft:dark_forest_hills, byg:polluted_lake,
                   byg:red_spruce_taiga, byg:ebony_woods, minecraft:badlands_plateau, byg:cypress_swamplands, byg:rocky_beach, minecraft:modified_jungle_edge, byg:red_rock_lowlands,
                   byg:evergreen_clearing, minecraft:giant_spruce_taiga_hills, byg:bog, minecraft:swamp_hills, byg:zelkova_clearing, byg:tropical_rainforest, minecraft:birch_forest,
                   byg:deciduous_forest, minecraft:mountains, byg:northern_forest, byg:jacaranda_forest_hills, byg:orchard, byg:snowy_evergreen_hills, byg:basalt_barrera,
                   minecraft:modified_jungle, byg:pumpkin_forest, byg:flowering_grove, byg:seasonal_deciduous_clearing, byg:shrublands]


[07Jan2021 20:50:10.114] [main/ERROR] [com.telepathicgrunt.blame.Blame/]: 
****************** Blame Report 1.7.6 ******************

 This ConfiguredFeature was found to be not registered. Look at the JSON info and try to
 find which mod it belongs to. Then go tell that mod owner to register their ConfiguredFeature
 as otherwise, it will break other mods or datapacks that registered their stuff.

 JSON info : {
  "config": {
    "feature": {
      "config": {},
      "type": "greekfantasy:small_nether_shrine"
    },
    "decorator": {
      "config": {
        "chance": 100
      },
      "type": "minecraft:chance"
    }
  },
  "type": "minecraft:decorated"
}

 Biome affected : [byg:sythian_torrids, byg:embur_bog, byg:magma_wastes, byg:withering_woods, byg:wailing_garth, byg:glowstone_gardens, minecraft:nether_wastes,
                   minecraft:warped_forest, minecraft:soul_sand_valley, minecraft:basalt_deltas, byg:warped_desert, byg:weeping_mire, byg:quartz_desert, minecraft:crimson_forest,
                   byg:subzero_hypogeal, byg:crimson_gardens, byg:brimstone_caverns]


[07Jan2021 20:50:10.114] [main/ERROR] [com.telepathicgrunt.blame.Blame/]: 
****************** Blame Report 1.7.6 ******************

 This ConfiguredFeature was found to be not registered. Look at the JSON info and try to
 find which mod it belongs to. Then go tell that mod owner to register their ConfiguredFeature
 as otherwise, it will break other mods or datapacks that registered their stuff.

 JSON info : {
  "config": {
    "feature": {
      "config": {
        "feature": {
          "config": {},
          "type": "greekfantasy:python_pit"
        },
        "decorator": {
          "config": {
            "outer": {
              "config": {},
              "type": "minecraft:square"
            },
            "inner": {
              "config": {},
              "type": "minecraft:heightmap"
            }
          },
          "type": "minecraft:decorated"
        }
      },
      "type": "minecraft:decorated"
    },
    "decorator": {
      "config": {
        "chance": 166
      },
      "type": "minecraft:chance"
    }
  },
  "type": "minecraft:decorated"
}

 Biome affected : [minecraft:bamboo_jungle, minecraft:bamboo_jungle_hills, byg:tropical_fungal_forest, byg:crag_gardens, byg:guiana_shield, minecraft:jungle, byg:tropical_rainforest,
                   minecraft:jungle_edge, minecraft:jungle_hills, minecraft:modified_jungle, minecraft:modified_jungle_edge]


[07Jan2021 20:50:10.115] [main/ERROR] [com.telepathicgrunt.blame.Blame/]: 
****************** Blame Report 1.7.6 ******************

 This ConfiguredFeature was found to be not registered. Look at the JSON info and try to
 find which mod it belongs to. Then go tell that mod owner to register their ConfiguredFeature
 as otherwise, it will break other mods or datapacks that registered their stuff.

 JSON info : {
  "config": {
    "feature": {
      "config": {
        "feature": {
          "config": {
            "feature": {
              "config": {
                "max_water_depth": 0,
                "ignore_vines": true,
                "heightmap": "MOTION_BLOCKING",
                "minimum_size": {
                  "limit": 0,
                  "lower_size": 0,
                  "upper_size": 0,
                  "min_clipped_height": 4,
                  "type": "minecraft:two_layers_feature_size"
                },
                "decorators": [],
                "trunk_provider": {
                  "state": {
                    "Properties": {
                      "axis": "y"
                    },
                    "Name": "greekfantasy:olive_log"
                  },
                  "type": "minecraft:simple_state_provider"
                },
                "leaves_provider": {
                  "state": {
                    "Properties": {
                      "waterlogged": "false",
                      "persistent": "false",
                      "distance": "7"
                    },
                    "Name": "greekfantasy:olive_leaves"
                  },
                  "type": "minecraft:simple_state_provider"
                },
                "foliage_placer": {
                  "radius": 2,
                  "offset": 4,
                  "height": 4,
                  "type": "minecraft:fancy_foliage_placer"
                },
                "trunk_placer": {
                  "base_height": 3,
                  "height_rand_a": 11,
                  "height_rand_b": 0,
                  "type": "minecraft:fancy_trunk_placer"
                }
              },
              "type": "greekfantasy:olive_tree"
            },
            "decorator": {
              "config": {},
              "type": "minecraft:spread_32_above"
            }
          },
          "type": "minecraft:decorated"
        },
        "decorator": {
          "config": {
            "count": 40
          },
          "type": "minecraft:count"
        }
      },
      "type": "minecraft:decorated"
    },
    "decorator": {
      "config": {
        "chance": 50
      },
      "type": "minecraft:chance"
    }
  },
  "type": "minecraft:decorated"
}

 Biome affected : [minecraft:snowy_taiga_mountains, minecraft:giant_tree_taiga_hills, byg:zelkova_forest, minecraft:taiga_hills, minecraft:taiga, minecraft:wooded_mountains, minecraft:jungle_edge,
                   minecraft:giant_tree_taiga, byg:jacaranda_forest, minecraft:snowy_taiga_hills, minecraft:dark_forest, byg:seasonal_forest, byg:evergreen_taiga, minecraft:forest,
                   minecraft:giant_spruce_taiga, byg:snowy_blue_taiga, byg:aspen_forest, byg:ancient_forest, minecraft:flower_forest, byg:seasonal_birch_forest, minecraft:snowy_taiga,
                   greekfantasy:olive_forest, byg:guiana_shield, byg:enchanted_forest, byg:seasonal_taiga, byg:weeping_witch_forest, minecraft:wooded_hills, byg:cherry_blossom_forest,
                   byg:glowshroom_bayou, byg:seasonal_deciduous_forest, byg:mangrove_marshes, byg:the_black_forest, byg:snowy_deciduous_forest, minecraft:tall_birch_forest, byg:snowy_coniferous_forest,
                   byg:araucaria_savanna, byg:maple_taiga, byg:snowy_evergreen_taiga, byg:bayou, byg:woodlands, minecraft:dark_forest_hills, minecraft:birch_forest_hills,
                   byg:ebony_woods, minecraft:taiga_mountains, byg:red_oak_forest, minecraft:giant_spruce_taiga_hills, byg:great_lakes, minecraft:birch_forest, byg:boreal_forest,
                   byg:deciduous_forest, byg:cika_woods, byg:orchard, byg:redwood_tropics, byg:blue_taiga, byg:coniferous_forest, minecraft:tall_birch_hills]


[07Jan2021 20:50:10.116] [main/ERROR] [com.telepathicgrunt.blame.Blame/]: 
****************** Blame Report 1.7.6 ******************

 This ConfiguredFeature was found to be not registered. Look at the JSON info and try to
 find which mod it belongs to. Then go tell that mod owner to register their ConfiguredFeature
 as otherwise, it will break other mods or datapacks that registered their stuff.

 JSON info : {
  "config": {
    "feature": {
      "config": {
        "feature": {
          "config": {},
          "type": "greekfantasy:harpy_nest"
        },
        "decorator": {
          "config": {
            "outer": {
              "config": {},
              "type": "minecraft:square"
            },
            "inner": {
              "config": {},
              "type": "minecraft:heightmap"
            }
          },
          "type": "minecraft:decorated"
        }
      },
      "type": "minecraft:decorated"
    },
    "decorator": {
      "config": {
        "chance": 83
      },
      "type": "minecraft:chance"
    }
  },
  "type": "minecraft:decorated"
}

 Biome affected : [byg:blue_taiga_hills, byg:snowy_coniferous_clearing, byg:evergreen_hills, byg:aspen_clearing, byg:twilight_valley_hills, byg:coniferous_forest_hills, minecraft:modified_gravelly_mountains,
                   byg:great_lake_isles, byg:marshlands, minecraft:swamp, byg:red_oak_forest_hills, byg:rainbow_beach, byg:jacaranda_forest, minecraft:desert_hills,
                   minecraft:dark_forest, byg:meadow, minecraft:jungle_hills, minecraft:mushroom_field_shore, byg:grassland_plateau, byg:wooded_red_rock_mountains, minecraft:giant_spruce_taiga,
                   byg:flowering_enchanted_grove, byg:red_desert, byg:deciduous_forest_hills, byg:tropical_islands, byg:tropical_fungal_forest, byg:cherry_blossom_clearing, byg:aspen_forest,
                   byg:enchanted_forest_hills, byg:frozen_lake, byg:zelkova_forest_hills, byg:white_beach, byg:guiana_clearing, byg:boreal_clearing, byg:fresh_water_lake,
                   greekfantasy:olive_forest, minecraft:mountain_edge, byg:enchanted_forest, byg:coniferous_clearing, minecraft:the_void, minecraft:wooded_hills, byg:glowing_ancient_forest,
                   byg:seasonal_taiga_hills, minecraft:stone_shore, byg:canyons, byg:boreal_forest_hills, minecraft:bamboo_jungle_hills, byg:mangrove_marshes, byg:amaranth_fields,
                   byg:blue_giant_taiga, minecraft:tall_birch_forest, minecraft:modified_wooded_badlands_plateau, byg:black_forest_hills, minecraft:eroded_badlands, byg:cika_mountains, byg:tropical_fungal_rainforest_hills,
                   byg:araucaria_savanna, byg:bayou, byg:rose_fields, byg:lush_red_desert, byg:woodlands, minecraft:plains, byg:snowy_coniferous_forest_hills,
                   minecraft:birch_forest_hills, minecraft:modified_badlands_plateau, byg:bluff_peaks, byg:maple_hills, byg:canyon_edge, byg:red_rock_mountains, byg:red_oak_forest,
                   byg:seasonal_giant_taiga, byg:dummy, byg:aspen_forest_hills, minecraft:jungle, byg:wooded_grassland_plateau, byg:boreal_forest, byg:cold_swamplands,
                   minecraft:badlands, byg:fungal_patch, byg:redwood_tropics, byg:seasonal_deciduous_forest_hills, minecraft:wooded_badlands_plateau, byg:seasonal_birch_forest_hills, byg:snowy_evergreen_clearing,
                   byg:redwood_clearing, byg:jacaranda_clearing, byg:snowy_blue_giant_taiga, byg:snowy_black_beach, minecraft:tall_birch_hills, byg:black_forest_clearing, byg:enchanted_grove,
                   minecraft:savanna_plateau, byg:deciduous_clearing, byg:redwood_mountains, minecraft:wooded_mountains, minecraft:jungle_edge, minecraft:savanna, byg:sierra_range,
                   byg:weeping_witch_clearing, byg:allium_fields, byg:snowy_rocky_black_beach, byg:bamboo_forest, byg:dunes, byg:prairie_clearing, byg:wooded_meadow,
                   minecraft:mushroom_fields, byg:flowering_meadow, byg:snowy_blue_taiga_hills, minecraft:forest, byg:alpine_foothills, byg:oasis, byg:coral_mangroves,
                   byg:flowering_ancient_forest, byg:forest_fault, minecraft:sunflower_plains, minecraft:gravelly_mountains, byg:ancient_forest, byg:sierra_valley, byg:snowy_deciduous_forest_hills,
                   byg:grove, byg:ebony_hills, minecraft:flower_forest, byg:tropical_rainforest_hills, minecraft:shattered_savanna_plateau, minecraft:desert, minecraft:beach,
                   byg:seasonal_forest_hills, byg:guiana_shield, byg:pointed_stone_forest, byg:red_rock_highlands, byg:mojave_desert, byg:cherry_blossom_forest, byg:glowshroom_bayou,
                   byg:red_desert_dunes, minecraft:shattered_savanna, byg:seasonal_deciduous_forest, byg:autumnal_valley, byg:stone_forest, byg:crag_gardens, byg:prairie,
                   minecraft:desert_lakes, byg:snowy_deciduous_clearing, byg:baobab_savanna, minecraft:bamboo_jungle, byg:araucaria_forest, minecraft:dark_forest_hills, byg:polluted_lake,
                   byg:red_spruce_taiga, byg:ebony_woods, minecraft:badlands_plateau, byg:cypress_swamplands, byg:rocky_beach, minecraft:modified_jungle_edge, byg:red_rock_lowlands,
                   byg:evergreen_clearing, minecraft:giant_spruce_taiga_hills, byg:bog, minecraft:swamp_hills, byg:zelkova_clearing, byg:tropical_rainforest, minecraft:birch_forest,
                   byg:deciduous_forest, minecraft:mountains, byg:northern_forest, byg:jacaranda_forest_hills, byg:orchard, byg:snowy_evergreen_hills, byg:basalt_barrera,
                   minecraft:modified_jungle, byg:pumpkin_forest, byg:flowering_grove, byg:seasonal_deciduous_clearing, byg:shrublands]

```

**Comment by skyjay1** [comment]
I thought I fixed this in #7 . What version of Greek Fantasy are you using?

**Comment by skyjay1** [comment]
Fixed in Greek Fantasy 16.4.8
